### DISCUSSION TITLE
[Question][Hw6] How is "efficiency score" computed?

### DISCUSSION THREAD

**Discussion by tncks0121** [prosyslab-classroom/cs524-program-analysis#311] [Q&A] [answered]
Name: 박수찬

Homework 4-6 에는 Worklist 알고리즘의 효율성을 측정하는 "효율성 점수"가 있습니다. 해당 점수의 계산 방식에 대한 힌트를 얻을 수 있을까요?

From Homework 4 to Homework 6, there is an eficiency score that measures efficiency of the worklist algorithm. May I get a hint on how that score is computed?

**Comment by goodtaeeun** [comment, marked as answer]
조교의 구현, 혹은 우수한 성능을 보인 이전 수강생들의 구현을 기준으로 메모리 오퍼레이션의 수를 계산합니다.
즉,  (모범 답안의 메모리 오퍼레이션 수) /  (제출된 답안의 메모리 오퍼레이션 수)으로 생각하시면 되겠습니다.

---

The efficiency score is computed based on the number of memory operations of the implementation of the TA or the past students.
The score is computed as follows: (The memory operation count of the standard implementation) / (The memory operation count of your implementation)
